tests/onnx/test_onnx_export_custom_module.py

```diff
@@ -60,3 +62,14 @@ def test_training(self):
                 do_constant_folding=do_constant_folding,
                 training=training,
             )
+        else:
+            # Expected to fail with opset_version < 12 on torch < 2.5
+            with self.assertRaises(Exception):
+                torch.onnx.export(
+                    sd,
+                    input,
+                    devnull,
+                    opset_version=11,
+                    do_constant_folding=do_constant_folding,
+                    training=training,
+                )
```

Comment:
this was failing on main for a while now (since torch 2.5 release)

Reply:
thanks!